lib/sprangular.rb

```diff
@@ -15,6 +15,8 @@
 require 'rails-assets-angular-bootstrap'
 require 'rails-assets-angular-strap'
 require 'rails-assets-angular-motion'
+require 'rails-assets-angular-translate'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.